css/styles.css

```diff
@@ -659,3 +659,286 @@ p#modal-close {
+    margin: 10px 0;
+    position: fixed;
+    top: -38%;
+    left: 50%;
+    transform: translate(-50%, 50%);
+    width: 80%;
+    border: 1px solid #fff;
+    z-index: 1000;
+    transition: display 1s linear;
+    background: #fff;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+    padding-bottom: 20px;
+    display: none;
+  }
+
+  .img-popup {
+    display: inline-block;
+    width: 100%;
+  }
+
+  .header-pop {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+    z-index: 1000;
+    width: 90%;
+  }
+
+  .img-popup-div {
+    width: 90%;
+    height: 320px;
+    margin: 0 auto;
+  }
+
+  .img-popup-div > img {
+    width: 100%;
+    height: 320px;
+    margin-top: 20px;
+  }
+
+  .close-btn {
+    margin: 0 10px;
+    z-index: 1000;
+    color: #67798e;
+    cursor: pointer;
+  }
+
+  p#modal-close {
+    font-size: 3.5rem;
+    font-size: 6;
+    color: #67798e;
+  }
+
+  .modal-description {
+    color: var(--color-headings);
+    font-size: 1.3rem;
+    line-height: 2.1rem;
+    display: inline-block;
+    width: 100%;
+    padding-right: 14px;
+    margin: 10px;
+  }
+
+  .card-specs {
+    display: flex;
+    width: 90%;
+    margin: 15px auto;
+  }
+
+  .languages-div {
+    display: flex;
+    flex-direction: column;
+    column-gap: 5px;
+    margin-bottom: 5px;
+    margin-top: 0;
+    justify-content: flex-start;
+  }
+
+  .description-div {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 90%;
+    margin: 15px auto;
+  }
+
+  .categories-list {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 30%;
+    margin: 8px;
+  }
+
+  .modal-btn button {
+    height: 30px;
+    text-align: center;
+    font-size: 10px;
+    margin: 5px 0;
+    padding: 4px 15px;
+    margin-right: 10px;
+  }
+
+  .modal-btn button > * {
+    margin: 4px;
+  }
+
+  .modal-para {
+    width: 70%;
+  }
+
+  .small-modal {
+    display: none;
+  }
+
+  .backend {
+    display: inline-block;
+  }
+
+  .modal-btn {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  a.mobile-btn.btn {
+    padding: 2rem 1.3em;
+    text-align: center;
+    width: 120px;
+    margin: 1.5rem 0.2rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  /* About Section */
+
+  .about-page {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    max-width: 100%;
+    align-items: center;
+    height: 644px;
+    background: var(--white);
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages li {
+    width: 122px;
+    height: 120px;
+    background-color: #f7f7f9;
+    margin-top: 0.8rem;
+    gap: 1.5rem;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+}
```

Comment:
- [ ] Nice work implementing the "About" section it looks really good :clap: However we must replicate the design as much as possible (e.g., font, colors, images, text, margins). Kindly add some spaces at the right and left of the about section content.

### Your design 
![Screenshot from 2023-03-31 15-02-20](https://user-images.githubusercontent.com/91142432/229162957-5f8829bd-e37d-475a-89a1-e6db5abf6fcb.png)
### What it supposes to look like
![Screenshot from 2023-03-31 15-25-40](https://user-images.githubusercontent.com/91142432/229163290-a1eef2fe-6c6f-49af-bc82-432e3a3cbb7d.png)